tests/units/test_device.py

```diff
@@ -102,6 +103,32 @@ def test__init__(self, device_data: dict[str, Any]) -> None:
             assert device.cache is not None
             assert device.cache_locks is not None
 
+    def test_supports(self) -> None:
+        """
+        Test if the supports() static method parses correctly the aioeapi.EapiCommandError exception
+        """
+        DATA: dict[str, Any] = {"host": "42.42.42.42", "username": "anta", "password": "anta"}
+        kwargs: dict[str, Any] = {
+            "name": "test.anta.ninja",
+            "enable": False,
+            "disable_cache": True,
+            "insecure": False,
+        }
+        device = AsyncEOSDevice(DATA["host"], DATA["username"], DATA["password"], **kwargs)
```

Comment:
why not using a fixture / mocked_device?

Reply:
does it still need to be tested at the AsyncEOSDevice level or could it be done at abstract AntaDevice one?